core/trino-main/src/main/java/io/trino/operator/MultiChannelGroupByHash.java

```diff
@@ -174,7 +174,8 @@ public long getEstimatedSize()
                 currentPageBuilder.getRetainedSizeInBytes() +
```

Comment:
try to limit the commit message subject line to 50 characters. see https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md

> Format Git commit messages
> When writing a Git commit message, follow these [guidelines](https://chris.beams.io/posts/git-commit/).